qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/plot_widget/fit_pane/plot_fit_pane_model.py

```diff
@@ -48,3 +49,11 @@ def _get_fit_label(workspace_name, index):
 
     def _is_guess_workspace(self, workspace_name):
         return self.context.guess_workspace_prefix in workspace_name
+
+    @staticmethod
+    def get_shade_lines(ws, index):
+        # need to copy the x data because if switch from points to hist data it causes errors
```

Comment:
Why does it only cause a problem with x data?

Reply:
because when we switch between histogram and point data its only the x values that change (bin boundaries or bin centres). The Y and E values are unchanged.